src/umple/actions/generate.ts

```diff
@@ -2,9 +2,10 @@ import * as vscode from "vscode";
 import * as path from "path";
 import { GENERATE_LANGS, umpleAPI } from "../umpleAPI";
 import { umpleLint } from "../../helpers/UmpleLintingProvider";
+import { Lint } from "../../commands/lint";
 
-export async function generate() {
-    let editor = vscode.window.activeTextEditor;
+export async function generate(this: any) {
```

Comment:
are we using `this` anywhere, also not a fan of using keywords as variable names.

Reply:
Changed and pushed. Would you please review that latest code?